docs/administering/active-directory.md

```diff
@@ -0,0 +1,304 @@
+This document explains how to integrate with Microsoft Active Directory (also known as AD).  This
+allows all users, or specific users within your Windows domain, to log into a Sandstorm server. The
+integration requires [Sandstorm for Work.](for-work.md)
+
+## AD & Sandstorm: Overview
+
+Sandstorm can rely on AD to authenticate users.
+
+ActiveDirectory exists as a cloud product called Windows Azure as well as a software product called
+Windows Server. Sandstorm supports both products.
+
+Sandstorm can be configured to use single sign-on in a configuration where Sandstorm never sees the
+user's passwords. This relies on Sandstorm's support for SAML 2.0.
+
+This document provides textual advice as well as a great deal of screenshots to allow you to proceed
+with confidence. If you have questions, please email support@sandstorm.io. We want to help you
+successfully set up Sandstorm!
+
+Note that you can also set up Sandstorm to integrate with Active Directory using LDAP bind to
+authenticate users. We recommend Active Directory Federation Services or Microsoft Azure AD Single
+Sign-On instead, which use SAML. SAML has the advantage that users never type passwords into
```

Comment:
You are in fact correct here. :/ This is just hard to read and sounds wrong because of Microsoft's tendency of naming schemes for things that are four words or longer.